paging.json

```diff
@@ -36,6 +37,33 @@
           }
         }
       }
+    },
+    "/{nextLink}": {
+      "post": {
+        "operationId": "examplePagingNext",
+        "description": "Subsequent post calls to the next link",
+        "parameters": [
```

Comment:
The parameters has to include path param `nextLink`. Include any other parameters you want passed to your subsequent `post` calls